genesis_zktrie.json

```diff
@@ -0,0 +1,73 @@
+{
+  "config": {
+    "chainId": 53077,
+    "homesteadBlock": 0,
+    "eip150Block": 0,
+    "eip150Hash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+    "eip155Block": 0,
+    "eip158Block": 0,
+    "byzantiumBlock": 0,
+    "constantinopleBlock": 0,
+    "petersburgBlock": 0,
+    "istanbulBlock": 0,
+    "berlinBlock": 0,
+    "londonBlock": 0,
+    "clique": {
+      "period": 15,
+      "epoch": 30000
+    },
+    "zktrie": true
+  },
+  "nonce": "0x0",
+  "timestamp": "0x61bc34a0",
+  "extraData": "0x0000000000000000000000000000000000000000000000000000000000000000221cb8Cd97223442754493A31184CF81F418E7280000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
+  "gasLimit": "0x6691b7",
+  "difficulty": "0x1",
+  "mixHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+  "coinbase": "0x0000000000000000000000000000000000000000",
+  "alloc": {
```

Comment:
Genesis account `4cb1ab63af5d8931ce09673ebd8ae2ce16fd6571` should in genesis file, or please add these accounts' keystore files in `alloc`.

Reply:
Ok. Now we provide two gensis objects (genesis.json and genesis_zktrie.json), both workable with the genesis keystore and no confusing data would be involved.